CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [gvk](gvk/) v0.0.1 (2023-07-07)
```

Comment:
Can we please call it `kubernetes-gvk`? `gvk` is so generic and assumes that everything we do here in this repo is about Kuebrnetes, which isn't really true.